ts/features/fci/screens/valid/FciDocumentPreviewScreen.tsx

```diff
@@ -39,23 +31,14 @@ export const FciDocumentPreviewScreen = (
   const [isError, setIsError] = React.useState(false);
   const docParamUrl = props.route.params.documentUrl ?? "";
   const [documentUrl, setDocumentUrl] = React.useState("");
-  const [showDocNavBar, setShowDocNavBar] = React.useState(false);
-  const [totalPages, setTotalPages] = React.useState(0);
-  const [currentPage, setCurrentPage] = React.useState(0);
-  const [currentDoc, setCurrentDoc] = React.useState(0);
-  const documents = useIOSelector(fciSignatureDetailDocumentsSelector);
   const fciDownloadPath = useIOSelector(fciDownloadPathSelector);
   const dispatch = useIODispatch();
 
   React.useEffect(() => {
```

Comment:
Do we really need this `useEffect` and `const [documentUrl, setDocumentUrl] = React.useState("");`? Cannot we directly use `const docParamUrl = props.route.params.documentUrl ?? "";`

Reply:
Good catch! ❤️ You are my hero! It's an old implementation used when messages with attachments weren't available for FCI.